tasks/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .common import logging_init_stream_handler, TimeoutException
-from .tasks import Build, RunPytest, RunWebuiTests, RunPytest2, RunPytest3
+from .tasks import Build, RunPytest, RunWebuiTests, RunPytest2, RunPytest3, RunMultiDomainPytest
```

Comment:
No need to define the new class RunMultiDomainPytest, see my comments in tasks/tasks.py